### PR TITLE
feat: bumping algokit utils, pyteal; removing algoexplorer as api pr…

### PR DESCRIPTION
**Pull request by aorumbayev**
## Proposed changes

- Bumping latest algokit utils v2 with dispenser support in order to be able to update algokit generators 
- Removing algoexplorer as api provide - no longer exist
- - Removing associated test

### CODE REVIEW

**Comment by aorumbayev** [comment]
@robdmoore can't seem to assign you in pr review - if possible please self assign for review

**Comment by robdmoore** [comment]
This looks good, but given it removes algoexplorer and does a major bump on utils it should probably at least bump minor version (not sure a `build` tag will do that)? I don't have permissions on this repo so can't merge it.

File: pyproject.toml [hunk 1]
```diff
@@ -8,23 +8,19 @@ readme = "README.md"
 homepage = "https://beaker.algo.xyz"
 repository = "https://github.com/algorand-devrel/beaker"
 keywords = ["pyteal", "algorand"]
-classifiers = [
-  "Development Status :: 2 - Pre-Alpha",
-]
-packages = [
-    { include = "beaker" },
-]
+classifiers = ["Development Status :: 2 - Pre-Alpha"]
```
Comment (achidlow) [inline pyproject.toml:11]: If we're gonna modify this bit anyway, it should be:
`Development Status :: 5 - Production/Stable`

File: pyproject.toml [hunk 2]
```diff
@@ -94,10 +104,10 @@ target-version = "py310"
 
 [tool.ruff.per-file-ignores]
 "tests/state_test.py" = ["ANN001", "ANN201", "ANN401"]
-"beaker/lib/*.py" = ["N802"]  # lowercase function names
+"beaker/lib/*.py" = ["N802"]                           # lowercase function names
```
Comment (achidlow) [inline pyproject.toml:107]: that's a lot of blank spaces lol

Comment (aorumbayev) [reply pyproject.toml]: @achidlow weird, editorconfig linter acting out

**Review by achidlow — APPROVED** [review]

**Pull request merged**